localstack-core/localstack/services/cloudformation/v2/provider.py

```diff
@@ -774,6 +784,7 @@ def create_stack(self, context: RequestContext, request: CreateStackInput) -> Cr
             account_id=context.account_id,
             region_name=context.region,
             request_payload=request,
+            tags=request.get("Tags") or [],
```

Comment:
```suggestion
            tags=request.get("Tags"),
```

nit: that or making the constructor stricter, but both with `or []` is a bit ambigous 🤷‍♂️